src/services/WorkspaceFlowRunsApi.ts

```diff
@@ -64,6 +65,13 @@ export class WorkspaceFlowRunsApi extends WorkspaceApi implements IWorkspaceFlow
     return mapper.map('FlowRunHistoryResponse', data, 'RunHistory')
   }
 
+  public async getFlowRunsAverageLateness(filter: FlowRunsFilter): Promise<string | null> {
+    const request = mapper.map('FlowRunsFilter', filter, 'FlowRunsFilterRequest')
+    const { data } = await this.post<number>('/lateness', request)
+
+    return data > 0 ? secondsToApproximateString(data) : null
```

Comment:
I would suggest this endpoint not return a formatted string. Doing so removes the possibility of doing any sort of math or calculation (or different formatting). Implementation should be responsible for formatting the seconds of lateness into a readable string.